graylog2-web-interface/src/views/pages/SearchPage.tsx

```diff
@@ -56,14 +57,15 @@ const SearchPageTitle = ({ children }: { children: React.ReactNode }) => {
 };
 
 const SearchPage = ({
-  children,
+  children = undefined,
   isNew,
   view: viewPromise,
   loadNewView: _loadNewView = defaultLoadNewView,
   loadView: _loadView = defaultLoadView,
   executionState: initialExecutionState,
-  searchResult,
-  forceSideBarPinned,
+  searchResult = undefined,
+  forceSideBarPinned = false,
+  skipNoStreamsCheck = false,
```

Comment:
For this version of the code, the default values should be added in the object below.

```

SearchPage.defaultProps = {
  loadNewView: defaultLoadNewView,
  loadView: defaultLoadView,
  executionState: SearchExecutionState.empty(),
  searchResult: undefined,
  forceSideBarPinned: false,
  skipNoStreamsCheck: false,
};